src/control/src/adapter_control/pacsim.cpp

```diff
@@ -66,12 +66,9 @@ void PacSimAdapter::finish() {
 
 void PacSimAdapter::publish_cmd(double acceleration, double steering) {
   auto steering_msg = pacsim::msg::StampedScalar();
-  auto acceleration_msg = pacsim::msg::Wheels();
+  auto acceleration_msg = pacsim::msg::StampedScalar();
 
-  // TODO: Convert values if necessary then fill the messages
-  //  CODE HERE
-  acceleration_msg.fl = acceleration_msg.fr = acceleration_msg.rl = acceleration_msg.rr =
```

Comment:
Having different values for each wheel could be quite useful in the future. I'll approve it because it's something easy to revert in the future